### ISSUE TITLE
Configuration file is not correct. Correcting

### ISSUE THREAD

**Issue by SettingDust** [InfornoFire/BackStab#4]
Version #: 1.16.5 1.1.1 Forge 36.2.22
Bug:
```
[17:57:10] [modloading-worker-3/WARN]: Configuration file H:\Minecraft\config\backstab-common.toml is not correct. Correcting
[17:57:10] [modloading-worker-3/WARN]: Incorrect key items.items was corrected from [backstab:wood_dagger, backstab:stone_dagger, backstab:iron_dagger, backstab:gold_dagger, backstab:diamond_dagger, backstab:netherite_dagger] to its default, [backstab:wood_dagger, backstab:stone_dagger, backstab:iron_dagger, backstab:gold_dagger, backstab:diamond_dagger, backstab:netherite_dagger]. 
[17:57:10] [modloading-worker-3/WARN]: Incorrect key entities.entities_blacklist was corrected from [minecraft:ender_dragon] to its default, [minecraft:ender_dragon]. 
```

![图片](https://user-images.githubusercontent.com/19922286/147382717-a6b19860-4a0c-4626-a711-3d8d55637fbb.png)
same content

**Comment by SettingDust** [comment]
https://github.com/MinecraftForge/MinecraftForge/issues/7669

**Comment by InfornoFire** [comment]
Fixed for 1.18.1, 1.16.5 no longer supported

**Comment by SettingDust** [comment]
> Fixed for 1.18.1, 1.16.5 no longer supported

Can u push it to github? So that, I can fix it by myself.
